grammar.js

```diff
@@ -121,11 +121,22 @@ module.exports = grammar({
       repeat($._blank),
     ),
 
+    // Language annotation for code blocks: only support core languages
+    language: ($) => choice(
```

Comment:
I think this could be wrapped with `token.immediate()` to enforce no whitespace between `>` and the language.

Reply:
But in the `codeblock`, not here, right?

Reply:
Whatever works. I'm still in the trial-and-error phase with these grammars.

Reply:
You're telling me? I have zero idea what I'm doing here, it's pure cargo cult ;)